src/utils/ExtensionLoader.js

```diff
@@ -65,10 +65,13 @@ define(function (require, exports, module) {
     // load the text and i18n modules.
     srcPath = srcPath.replace(/\/test$/, "/src"); // convert from "test" to "src"
 
-    var globalConfig = {
-            "text" : srcPath + "/thirdparty/text/text",
-            "i18n" : srcPath + "/thirdparty/i18n/i18n"
-        };
+    // Retrieve the global paths
+    var globalPaths = brackets.getGlobalRequireJSConfig().paths;
+
+    // Convert the relative paths to absolute
+    Object.keys(globalPaths).forEach(function (key) {
+        globalPaths[key] = PathUtils.makePathAbsolute(srcPath + "/" + globalPaths[key]);
+    });
```

Comment:
I hope not, but if we'll find problems with this approach we'll have to dupe again this configuration:
https://github.com/adobe/brackets/blob/bf9b07d8e7b238a0f7f5392c8460ad34b63ee12c/src/main.js#L29
right?